lightning/src/ln/channelmanager.rs

```diff
@@ -6581,6 +6619,14 @@ where
 						let mut chan = remove_channel_phase!(self, chan_phase_entry);
 						finish_shutdown = Some(chan.context_mut().force_shutdown(false, ClosureReason::CounterpartyCoopClosedUnfundedChannel));
 					},
+					// TODO(dual_funding): Combine this match arm with above.
```

Comment:
This arm is very similar to the arm above (minor difference in logger context handling, maybe should be the same). It looks like the two arms could be unified.

Reply:
Currently because of the cfg flag, it's not possible (hence the TODO). But you're right for when the flag is eventually removed!

I'll add "once #[cfg(dual_funding)] is removed" to the TODO to make it clearer.